apps/judicial-system/web-e2e-regression/src/integration/Prosecutor/Indictments/defendant.spec.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  CREATE_INDICTMENT_ROUTE,
+  INDICTMENTS_PROCESSING_ROUTE,
+} from '@island.is/judicial-system/consts'
+import { CaseTransition } from '@island.is/judicial-system/types'
+
+import { transitionCase } from '../../../utils'
+
+describe('Create indictment', () => {
+  let caseId = ''
+
+  before(() => {
+    cy.visit('http://localhost:4200/api/auth/login?nationalId=0000000009')
+    cy.intercept(
+      'GET',
+      '**/api/nationalRegistry/getPersonByNationalId**',
+      (req) => {
+        req.reply({ fixture: 'nationalRegistryPersonResponse' })
+      },
+    ).as('getPersonByNationalId')
+  })
+
+  after(() => {
+    transitionCase(caseId, CaseTransition.DELETE)
+  })
+
+  it('should be able to create a case', () => {
+    cy.getByTestid('createCaseDropdown').click()
+    cy.get(`a[href="${CREATE_INDICTMENT_ROUTE}"]`).click()
+    cy.url().should('contain', CREATE_INDICTMENT_ROUTE)
+
+    cy.get('#policeCaseNumbers').type('0').type('{enter}')
+    cy.getByTestid('policeCaseNumbers-list').children().should('have.length', 0)
```

Comment:
Just a heads up. This will need to be updated because of https://github.com/island-is/island.is/pull/8450